calico-cloud_versioned_docs/version-3.17/variables.js

```diff
@@ -1,22 +1,22 @@
 const releases = require('./releases.json');
 
 const variables = {
-  releaseTitle: 'v3.16.1',
+  releaseTitle: 'v3.17.1',
   prodname: 'Calico Cloud',
   prodnamedash: 'calico-cloud',
   baseUrl: '/calico-cloud',
   filesUrl: 'https://docs.calicocloud.io',
-  filesUrl_CE: 'https://downloads.tigera.io/ee/v3.16.1',
+  filesUrl_CE: 'https://downloads.tigera.io/ee/v3.17.1',
   tutorialFilesURL: 'https://docs.tigera.io/files',
   prodnameWindows: 'Calico Enterprise for Windows',
   rootDirWindows: 'C:\\TigeraCalico',
   nodecontainer: 'cnx-node',
   noderunning: 'calico-node',
-  cloudversion: 'v3.16.1-5',
-  clouddownloadurl: 'https://installer.calicocloud.io/manifests/v3.16.1-5',
+  cloudversion: 'v3.17.1-5',
+  clouddownloadurl: 'https://installer.calicocloud.io/manifests/v3.17.1-5',
   clouddownloadbase: 'https://installer.calicocloud.io',
   cloudoperatorimage: 'us-docker.pkg.dev/tigera-prod-01/calico-cloud-public/cc-operator',
-  imageassuranceversion: 'v1.5.6',
+  imageassuranceversion: 'v1.7.3',
```

Comment:
Where did you get this number from? I think it should be v1.8.4

Reply:
Updated.